central/src/components/CreateQuestion.jsx

```diff
@@ -97,12 +96,24 @@ export default function QuestionForm({ updateQuestion, question: initialState, g
       window.alert("Please enter a question");
       return;
     }
-
+    if (question.choices[0].text == null || question.choices[0].text === "") {
+      window.alert("Please enter a correct answer")
+      return;
+    }
+    if (question.instructions == null || question.instructions.filter(step => step !== "").length < 1) {
+      window.alert("Please provide at least one step for the correct answer explanation");
+      return;
+    }
+    for (let choiceI = 1; choiceI < (question.choices).length; choiceI++) {
```

Comment:
It's usually better not to shorten, like `choiceI` vs. `choiceIndex` or just go with `index`. `index` can be shorten to `i` or `idx` as it's very common and used often, but when it's appendix, you can shorten it to `choiceIdx` for example but not `I` as latter is not common